src/borg/archiver.py

```diff
@@ -2285,12 +2285,16 @@ def build_parser(self, prog=None):
 
     @staticmethod
     def add_archives_filters_args(subparser):
+        filters_group = subparser.add_argument_group('filters', 'Archive filters can be applied on repository targets.')
+        filters_group.add_argument('-P', '--prefix', dest='prefix', type=prefix_spec, default='',
+                               help='only consider archive names starting with this prefix')
```

Comment:
at the other places (old code), there is no default, why do you have one here?

please note that a empty string prefix matches all archives, which might be dangerous if the operation is "delete".